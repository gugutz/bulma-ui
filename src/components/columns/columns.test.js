import React from 'react'
import renderer from 'react-test-renderer'
import Columns from './columns'

describe('<Columns /> component', () => {
  it('should render correctly', () => {
    const tree = renderer
      .create(
        <Columns>
          <div>Some content</div>
        </Columns>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('should support isCentered prop', () => {
    const tree = renderer
      .create(
        <Columns isCentered>
          <div>Some content</div>
        </Columns>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
