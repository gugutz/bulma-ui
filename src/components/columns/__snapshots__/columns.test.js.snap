// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Columns /> component should render correctly 1`] = `
<div
  className="columns"
>
  <div>
    Some content
  </div>
</div>
`;

exports[`<Columns /> component should support isCentered prop 1`] = `
<div
  className="columns is-centered"
>
  <div>
    Some content
  </div>
</div>
`;
